### PR TITLE
Make boomer execution abortable with SIGINT.

### PR DESCRIPTION
**Pull request by alevinval**
I find myself using boom to ramp up connections on an SSE stream.
The stream never finishes and I can't stop boomer with keyboard interrupts.
This patch lets me kill boom with keyboard interrupts vastly improving my boom experience.

### CODE REVIEW

**Comment by ottob** [comment]
:+1: thanks for adding this.

File: boom.go [hunk 1]
```diff
@@ -197,3 +204,18 @@ func parseInputWithRegexp(input, regx string) ([]string, error) {
...
+	finished := make(chan bool, 1)
+	go func() {
+		boomer.Run()
+		finished <- true
+	}()
+	return finished
+}
+
+func sysExit() <-chan os.Signal {
+	ch := make(chan os.Signal)
```
Comment (toin0u) [inline boom.go]: I think we need a buffer to not miss the signal, isn't it?

Comment (alevinval) [inline boom.go]: You are right. At least a buffer of size 1.

Comment (toin0u) [inline boom.go]: +1

**Pull request closed without merging**